src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -302,35 +282,70 @@ export class BaseEditCourseForm extends React.Component {
               <FieldLabel id="number" text="Number" className="mb-2" />
               <div className="mb-3">{number}</div>
             </div>
-            <Field
-              name="mode"
-              component={RenderSelectField}
-              label={
-                <FieldLabel
-                  id="mode.label"
-                  text="Enrollment track"
-                  helpText={enrollmentHelp}
-                  extraText="Cannot edit after submission"
+            {type ? (
+              <React.Fragment>
```

Comment:
Should we be sharing more code between the edit form and the create form? Seems like we have a lot of dup'd lines.

Reply:
I'd have to look back into it, but if I remember correctly, there are some small differences between the two forms that make it hard to reuse code.